### DISCUSSION TITLE
Is v2 stable? Will artillery quick make to v2?

### DISCUSSION THREAD

**Discussion by PavloPoliakov** [artilleryio/artillery#1271] [Artillery]
Good day, I'm translating [this article](https://blog.appsignal.com/2021/11/10/a-guide-to-load-testing-nodejs-apis-with-artillery.html) and met some issues, which are blocking `artillery` popularisation currently.

The article is fresh, but there is goes about `artillery@1`. Now `v2` is mainstream.

Have several questions about that:
1. I was not able to find a change log or announsement about the `v2` release, namely [here](https://github.com/artilleryio/artillery/releases).
  I have found migration guide [here](https://www.artillery.io/docs/resources/core/v2-migration-guide), there is nothing about `artillery quick` and the page says that `v2` is under active development.
  While [here](https://www.artillery.io/docs/guides/getting-started/installing-artillery#stable-release), in "getting started", it says that `v2` is a stable release.
  Who to believe? Which version people shall use today?
2. I have found ([here](https://www.artillery.io/docs/guides/guides/command-line)), that `artillery quick` is only available in `v1`. Are there plans to include it to `v2` as well later? It's mentioned in the original article and I wonder if that shall be in the translated one.
3. On this page https://www.artillery.io/docs , there is a link to `Discord`, it is broken. What is the correct one?

Thanks for your work, looking forward to the answers.

Regards,

**Comment by PavloPoliakov** [comment]
Good day @hassy @flentini , please, consider answering my questions.

**Comment by hassy** [comment]
hi @PavelPolyakov 👋 apologies for the radio silence, just getting back into the swing of things after the holiday period! 🎄 

Things are a bit in flux with v2 at the moment. It's been upgraded from `dev` to `latest` channel on npm so that's what someone installing Artillery from scratch will get. Any existing projects that pin to v1 still get that version (and it will still get security updates and critical bug fixes, it has not been deprecated). We'll update the docs to reflect that, thank you for flagging it up.

`artillery quick` will make its way over to v2 as well, but we haven't got round to it yet.

Will fix the Discord link in the docs too.
